components/clarinet-cli/src/devnet/package.rs

```diff
@@ -0,0 +1,91 @@
+use std::fs::{read_to_string, File};
+use std::io::{self, Write};
+use std::process;
+
+use clarinet_deployments::types::DeploymentSpecificationFile;
+use clarinet_files::{FileLocation, NetworkManifestFile, ProjectManifestFile};
+
+#[derive(Serialize, Deserialize, Debug)]
+struct ConfigurationPackage {
+    deployment_plan: DeploymentSpecificationFile,
+    devnet_config: NetworkManifestFile,
+    clarinet_config: ProjectManifestFile,
+}
+
+fn get_devnet_config() -> Result<NetworkManifestFile, io::Error> {
+    let file_content = read_to_string("./settings/Devnet.toml")?;
+
+    let devnet_config: NetworkManifestFile = match toml::from_str(&file_content) {
+        Ok(data) => data,
+        Err(err) => {
+            println!("Unable to load data from Devnet.toml file: {}", err);
+            process::exit(1);
+        }
+    };
+
+    Ok(devnet_config)
+}
+
+fn get_clarinet_config(manifest_location: FileLocation) -> Result<ProjectManifestFile, io::Error> {
+    let clarinet_config_content = read_to_string(manifest_location.to_string())?;
+
+    let clarinet_config: ProjectManifestFile = match toml::from_str(&clarinet_config_content) {
+        Ok(data) => data,
+        Err(err) => {
+            println!("Unable to load data from Clarinet.toml file: {}", err);
+            process::exit(1);
+        }
+    };
+
+    Ok(clarinet_config)
+}
+
+fn get_deployment_plan() -> Result<DeploymentSpecificationFile, io::Error> {
+    let deployment_spec_file = File::open("./deployments/default.simnet-plan.yaml")?;
+    let deployment_plan: DeploymentSpecificationFile =
```

Comment:
I think we will want to use `DeploymentSpecification`'s ` `from_config_file` function instead. This will give us the contract source code included in the deployment plan.

Also, this function requires a `deployment_location` that I think we can get from the user rather than a hardcoded path. There also is the `get_default_deployment_path` function for if the user doesn't provide the path.

Reply:
Done.